src/current/_includes/v24.1/cdc/disable-replication-ttl.md

```diff
@@ -0,0 +1,29 @@
+{% include_cached new-in.html version="v24.1" %} Use the `ttl_disable_changefeed_replication` table storage parameter to prevent changefeeds from sending messages on a per-table basis for changes caused by row-level TTL jobs. Include the storage parameter when you create or alter the table. For example:
```

Comment:
> from sending messages on a per-table basis for changes caused by row-level TTL jobs

I think we can be a bit more explicit and say that setting this option will mean there will be no changefeed events emitted for the `DELETE`s issued by row-level TTL (those are the only kinds of updates issued by row-level TTL).

Reply:
Done!